guide/sections/part2/global-services.adoc

```diff
@@ -218,3 +222,38 @@ wis2-gdc is managed as a free and open source project.  Source code, issue track
 * As a convention Global Monitor centre-id will be ``tld-{centre-name}-global-monitor``.
 
  The main task of the Global Monitor is to regularly query the provided metrics from the relevant WIS2 entities, aggregate and process the data and then provide the results to the end user in a suitable presentation.
+
+==== Global Replay
+
+===== Technical considerations
+
+* The Global Replay provides Global Services and Data Consumers with a mechanism to search and query for notification messages of interest.
```

Comment:
Do we want to open the GR to "Joe public" ? It is a service for operations.

Reply:
Yes.  While it is a service for operations it will also be valuable for users over time.